src/lib.rs

```diff
@@ -235,6 +235,15 @@ The minimum supported Rust version is `1.45.0`.
 #![doc(html_root_url = "https://docs.rs/glam/0.17.3")]
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(target_arch = "spirv", feature(asm, register_attr, repr_simd))]
+#![deny(
+    rust_2018_compatibility,
+    rust_2018_idioms,
+    future_incompatible,
+    nonstandard_style
+)]
+// Wait until this bug is fix and float cmp to zero don't report a warning.
+// https://github.com/rust-lang/rust-clippy/issues/3804
+#![allow(clippy::float_cmp)]
```

Comment:
Is there more than one of these generating warnings? If not it might be better to allow the specific line instead of allowing at the module level.

Reply:
I will put it in the scalar.rs file since this is the only place where there is a wrong warning. But rust complain about not using an allow in a macro so i can't put in in the specific line.

Reply:
Ah I see